tools/uswds-component-library-figma.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: default
+title: USWDS Component Library in Figma
+nav: Tools
+permalink: /tools/uswds-component-library-figma
+sidenav: tools-side-nav
+---
+
+# {{ page.title }}
+
+The [USWDS component library](https://www.figma.com/community/file/836611771720754351) is available and free to use through the Figma Community. Created by the [Truss.works team](https://truss.works/), it's a great place to start incorporating components into your designs.
+
+## Getting started
+
+### Duplicating the USWDS component library
+
+To view and duplicate files from the Figma Community, login or create a [Figma account](https://www.figma.com/).
+
+1. Navigate to the community tab in the left-hand side navigation
+1. Search for "USWDS" in the search bar
+1. Open the USWDS file by Truss
+1. Duplicate the file
+
+![GIF demonstrating how to navigate through Figma's community page to duplicate the U.S. Web Design System library created by Truss.]({{ site.url }}/{{site.baseurl}}/assets/img/uswds-component-library-getting-started.gif)
+
+### Publishing the library
+
+You've successfully duplicated the library! Now you can publish this file and apply it across all your project files.
+
+![GIF demonstrating how to navigate through Figma's community page to duplicate the U.S. Web Design System library created by Truss.]({{ site.url }}/{{site.baseurl}}/assets/img/uswds-component-library-publishing-file.gif)
+
+## Start crafting designs 
+
+Not sure where to start or need inspiration? 
+
+We've gone ahead and created some page templates using the USWDS components library. Check them out [here](#)!
```

Comment:
Not best practice for accessibility to link "here" text only. It's better for the linked text to describe (or name) the thing you're linking to. We can keep the exclamatory enthusiasm, but just change the link text.